versions/2.0.0/asyncapi.md

```diff
@@ -1105,6 +1121,15 @@ correlationId:
   location: $message.header#/correlationId
 traits:
   - $ref: "#/components/messageTraits/commonHeaders"
+examples:
+  - headers:
+      correlationId: my-correlation-id
+      applicationInstanceId: myInstanceId
+    payload:
+      user:
+        someUserKey: someUserValue
+      signup:
+        someSignupKey: someSignupValue
```

Comment:
Is it a good idea to show example of not `Schema object`?

Reply:
This shouldn't be a Schema object. It's an example of what you will transfer down the wire not an AsyncAPI example.